packages/api/src/repository.js

```diff
@@ -55,7 +55,7 @@ class Repository {
     return data.ticket;
   }
 
-  async allowedToPerformTicket (action, permission) {
```

Comment:
good catch 👍  all that `async` is, is a short hand for:

```
allowedToPerformTicket(() => {
  return new Promise((resolve, reject)) => {

  })
})
```

It's good to avoid specifying async unless it's needed.